internal/helpers.go

```diff
@@ -140,12 +141,12 @@ func GetSrcIndexFromId(indexes []schema.Index, indexId string) (schema.Index, er
 func ComputeUsedNames(conv *Conv) map[string]bool {
 	usedNames := make(map[string]bool)
 	for _, table := range conv.SpSchema {
-		usedNames[table.Name] = true
+		usedNames[strings.ToLower(table.Name)] = true
```

Comment:
is this being done throughout the code ? How was it working before this PR as the older code snippet does not move to lower.

Reply:
made the changes to do it throughout the code. It was buggy before this PR. It had .lower() missing at some places